.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+node_js:
+  - lts/*
+
+cache: yarn
+
+before_deploy:
+  - yarn install
```

Comment:
I think yarn install will be called by the install step, so it shouldn't be needed explicitly?

Reply:
Not sure, probably not because I've overridden default deploy script